### PR TITLE
Set Scrutinizer external coverage timeout

### PR DESCRIPTION
**Pull request by ddeboer**
See FriendsOfSymfony/FOSHttpCacheBundle#216.

### CODE REVIEW

**Comment by stof** [comment]
I suggest moving the coverage job higher in the matrix so that it starts earlier, to give it a chance to block scrutinizer for less time

**Comment by ddeboer** [comment]
Will try that, @stof. I think the `matrix:` builds always run later than the `php:` ones, or should I just move the `matrix` block up?

**Comment by stof** [comment]
no, the jobs included manually are indeed added in the matrix after the autogenerated ones. But as more than half our jobs are added in the matrix manually, it will still be better than having it last

File: .travis.yml [hunk 1]
```diff
@@ -32,10 +29,13 @@ matrix:
...
-      env:
-        - PHPUNIT_FLAGS="--coverage-clover=coverage.clover"
-        - COVERAGE=true
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 7.0
+  - hhvm
```
Comment (stof) [inline .travis.yml]: changing this is useless (the order of top-level keys is meaningless). and keeping the list of PHP versions on top near the definition of the language makes more sense IMO

Comment (ddeboer) [inline .travis.yml]: That’s what I suspected. Moved it back.

**Pull request merged**